src/components/Profile/index.tsx

```diff
@@ -2,7 +2,7 @@
 import Icon from '../Global/Icon'
 
 import { createAvatar } from '@dicebear/core'
-import { avataaarsNeutral } from '@dicebear/collection'
+import { funEmoji } from '@dicebear/collection'
```

Comment:
TODO: let's use identicons from dicebear. The problem with a specific artstyle like these cartoon avatars is that they clash a lot with current peanut colors and it makes for an uncohesive whole

![image](https://github.com/user-attachments/assets/bc059b11-b3f1-457b-82fc-7dce1a06c8ee)